components.json

```diff
@@ -12,6 +12,6 @@
   },
   "aliases": {
     "components": "@/components",
-    "utils": "@/lib/utils"
+    "utils": "@/utils/utils"
```

Comment:
Why was this changed?

Reply:
when i installed and tried to implement shad-cn/ui, it wasn't working properly. Chris had me change it to this.

Reply:
@chris-nowicki explain yourself please (for my knowledge more than anything). I've installed components via the CLI without a problem before.

Reply:
@ryandotfurrer this is because the utils file has been moved to /utils/utils.ts so updated the config for shadcn so it knows that is where the CN function is.

Reply:
@chris-nowicki okay cool beans; once this is merged everyone on the team will have to pull from develop to ensure they have no issues.

Reply:
Word. Agreed.